pkg/kubectl/resource/visitor.go

```diff
@@ -222,7 +222,7 @@ func (v *PathVisitor) Visit(fn VisitorFunc) error {
 		if !v.IgnoreErrors {
 			return err
 		}
-		fmt.Fprintf(os.Stderr, "error: unable to load file %q: %v\n", v.Path, err)
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
Why is this path removed but not the one in the error from line 218 or 215? It seems like those would cause the same formatting bug? How did you decide which ones to use?